### PR TITLE
Fix breadcrumbs logger

### PR DESCRIPTION
**Pull request by 0atman**
#### What problem does the pull request solve?

no logs appear in sentry

#### Checklist

- [ ] I've used the pull request template
- [ ] I've linked this PR to the relevant issue (if mission work)
- [ ] I've written unit tests for these changes (if code change)
- [ ] I've updated the documentation in (If any documentation requires updating)
    - [ ] README.md
    - [ ] Elsewhere (please link)

### CODE REVIEW

File: config/initializers/sentry.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Sentry.init do |config|
   config.dsn = ENV["SENTRY_DSN"]
-  config.breadcrumbs_logger = %i[sentry_logger http_logger]
+  config.breadcrumbs_logger = [:active_support_logger, :http_logger]
```
Comment (0atman) [inline config/initializers/sentry.rb:3]: @DavidBiddle @thomasiles can someone smarter than me tell me what this %i jibberjabber means? :laughing:

**Review by DavidBiddle — APPROVED** [review]
Looks good to me 🎉

**Review by thomasiles — APPROVED** [review]
Nice one!

File: Gemfile.lock [hunk 2]
```diff
@@ -183,8 +183,6 @@ GEM
       net-protocol
       timeout
     nio4r (2.5.8)
-    nokogiri (1.13.7-x86_64-darwin)
-      racc (~> 1.4)
     nokogiri (1.13.7-x86_64-linux)
       racc (~> 1.4)
```
Comment (0atman) [inline Gemfile.lock:187]: oh gods this doesn't look annoying AT ALL

**Comment by 0atman** [comment]
If anyone works on the runner before Monday, do merge this in to get the new version of ruby :ok_hand:

**Pull request merged**